package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "movieproject",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+     "dev":"nodemon index.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@types/express": "^4.17.17",
```

Comment:
Make sure you are using the libraries in the correct place.
https://www.geeksforgeeks.org/difference-between-dependencies-devdependencies-and-peerdependencies/

In this case, `@types/express` has to be a devDependency since it only helps us during development.